src/sentry/auth/idpmigration.py

```diff
@@ -15,8 +46,21 @@ def create_verification_key(user: User, org: Organization, email: str) -> None:
     :param org: the organization whose SSO provider is being used
     :param email: the email address associated with the SSO identity
```

Comment:
looks like you need to update these comments to include the new identity_id param. I would prefer deleting the parameter docs in general, but that is a stylistic preference.